brave/brave-core
# Adds uphold server handler

Resolves https://github.com/brave/brave-browser/issues/11286

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (unit, browser, security tests) on

## Test Plan:
- connect to brand new kyc account and make sure that card is created
- connect to existing kyc account and make sure that funds are transferred to correct card
- do tip where you have kyc account and no VG funds

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.
